exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java

```diff
@@ -184,25 +183,26 @@ public void testAllScalarTypes() throws Exception {
 
     try {
       // read all of the types with the complex reader
-      test(String.format("alter session set %s = true", ExecConstants.PARQUET_NEW_RECORD_READER));
+      alterSession(ExecConstants.PARQUET_NEW_RECORD_READER, true);
```

Comment:
Paul,

- I noticed the key PARQUET_NEW_RECORD_READER is erroneous
- There are currently two readers
   o The old one is used when nested data is used as it can handle all parquet use-cases
   o The new reader only deals with Flat parquet data types
- We might want to rename the keys as the new reader cannot always be substituted with the old one

Reply:
Good point. Here, I was not changing the semantics of the test; rather just ensuring that the tests are deterministic in doing what they already did. Presumably the test author knew which reader to use for which test. In fact, the fact that the tests now pass indicates that the proper reader is used (or, at least, the wrong reader is not used...)